samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -18,10 +18,13 @@
 #include "app_dfu.h"
 #include "app_factory_reset.h"
 #include "app_fp_adv.h"
-#include "app_motion_detector.h"
 #include "app_ring.h"
 #include "app_ui.h"
 
+#if CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR
```

Comment:
Does it build without warning when the `CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` is disabled?

I am wondering if the compiler is fine with the missing API calls declarations before it finds out that the `app_motion_detector_init` is not needed

Reply:
It builds with a warning, but I decided to leave it to avoid ifdef

Reply:
I think we have a ticket to get rid of the CMake zephyr library definitions as part of the motion detector CMake file. Perhaps the ticket scope could be extended or the following work scope could be defined as part of a separate ticket:

Move the public headers (typically placed in the /include folder) from the standalone libraries of this application:

    - battery
    - motion_detector
    - ui

Reply:
Okay, I will add that to the ticket.